docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -21,5 +21,5 @@ Person *--> Address
 Person *--> Remark
 Person *--> Birthday
 Person *--> MoneyOwed
-Person *--> "*" Day
```

Comment:
Why did you change the aggregate to normal arrow ah?

Reply:
I was taking reference from the bettermodelclassdiagram where they used a normal arrow between person and tags, and I realised that Day has the same type of association with person.

I changed the composition to normal arrow, but now that you mention it, do you think using a hollow diamond to represent aggregate is better?